.travis.yml

```diff
@@ -5,8 +5,6 @@ branches:
   - master
   - /^parity-.*$/
 
-cache: cargo
```

Comment:
I've disabled cache for tests, as it seems that retrieving the cache causes builds to time out.

Reply:
See https://github.com/paritytech/wasmi/pull/213/files

Reply:
Thanks a lot!